### ISSUE TITLE
helm chart defaults to using s3

### ISSUE THREAD

**Issue by justincely** [DecipherNow/helm-charts#279]

**Comment by justincely** [comment]
@tilleryd thoughts on that?

**Comment by tilleryd** [comment]
@justincely Agreed, the scales will balance out soon. We've just been optimizing for production deployments recently.

**Comment by whenhellfreezes** [comment]
@justincely @tilleryd If you use the makefile targets then make credentials provides a prompt for this choice and fills in the proper credentials if given or sets S3 to disabled. Does this satisfy this issue being closed?
